manifests/common/apps/jupyter/jupyter-app.yaml

```diff
@@ -203,25 +202,25 @@ spec:
 
       If you see **Error 403: Forbidden**, you are not in the default user group or the default administrator group for OpenShift AI. Contact your administrator so that they can add you to the correct group using [Adding users for Red Hat OpenShift AI](https://docs.redhat.com/en/documentation/red_hat_openshift_ai_cloud_service/1/html/managing_openshift_ai/managing-users-and-groups#adding-users-to-user-groups_managing-rhoai).
 
-    3. Start a notebook server.
+    3. Start a workbench.
 
       This is not required if you have previously launched Jupyter.
 
-      i. Select the **Notebook image** to use for your server.
+      i. Select the **Notebook image** to use for your workbench.
 
       ii. If the notebook image contains multiple versions, select the version of the notebook image from the **Versions** section.
 
-      iii. Select the **Container size** for your server.
+      iii. Select the **Container size** for your workbench.
 
-      iv. Optional: Select the **Number of GPUs** (graphics processing units) for your server.
+      iv. Optional: Select the **Number of GPUs** (graphics processing units) for your workbench.
```

Comment:
Same as above: We call GPUs "Accelerators" elsewhere in the product. If out of scope, please ignore this comment, but if not it may make sense to change it here.